contracts/explorations/GenArt721CoreV3_Explorations.sol

```diff
@@ -237,7 +237,7 @@ contract GenArt721CoreV3_Explorations is
     /// version & type of this core contract
     /// coreVersion is updated from Flagship V3 core due to minor changes
     /// implemented in the Explorations version of the contract.
-    string public constant coreVersion = "v3.0.3";
+    string public constant coreVersion = "v3.1.0";
```

Comment:
nit: I think would be good to make this v3.1.1 do differentiate from flagship

Reply:
Great call out – done!